app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -27,7 +27,7 @@ message SchedulerConfig {
 }
 
 // A unit of work for infrastructure to run.
-// Next ID: 17
+// Next ID: 18
```

Comment:
Should this not be 19? Also, why do we need this?

Reply:
Ah, it should be. This looks like a relic of ye'olden times.  I'll just back this out and we can have a single PR updating this.